### ISSUE TITLE
--die-on no longer dies

### ISSUE THREAD

**Issue by domenic** [speced/bikeshed#2769]
If you check out https://github.com/whatwg/xhr/tree/a2c40a0be1af035c093b07026802d1a327a74091 and run

```
bikeshed spec xhr.bs xhr.spec.whatwg.org/index.html --die-on=everything --md-Text-Macro="COMMIT-SHA FOO"
```

the `xhr.spec.whatwg.org/index.html` file will still be output.

**Comment by domenic** [comment]
`Die On:` metadata works.

**Comment by tabatkins** [comment]
Yup, when I added the metadata version, I didn't manually pipe the existing command-line flags into the metadata infrastructure, so they couldn't override. Fixed in cb9a5d95f, sorry about that.
